libs/vkd3d/resource.c

```diff
@@ -3481,29 +3482,37 @@ bool vkd3d_create_texture_view(struct d3d12_device *device, const struct vkd3d_t
     view_desc.subresourceRange.baseArrayLayer = desc->layer_idx;
     view_desc.subresourceRange.layerCount = desc->layer_count;
 
-    if (desc->miplevel_clamp != 0.0f)
-        FIXME_ONCE("Cannot handle MinResourceLOD clamp of %f correctly.\n", desc->miplevel_clamp);
-
-    /* This is not correct, but it's the best we can do with existing API.
-     * It should at least avoid a scenario where implicit LOD fetches from invalid levels.
-     * TODO: We will need an extension with vkCreateImageView pNext specifying minLODClamp.
-     * It will be trivial to add in RADV at least ... */
-    if (desc->miplevel_clamp >= 1.0f)
+    if (device->device_info.image_view_min_lod_features.minLod)
     {
-        uint32_t clamp_base_level;
-        uint32_t new_base_level;
-        uint32_t end_level;
+        min_lod_desc.sType = VK_STRUCTURE_TYPE_IMAGE_VIEW_MIN_LOD_CREATE_INFO_EXT;
+        min_lod_desc.pNext = NULL;
+        min_lod_desc.minLod = desc->miplevel_clamp;
```

Comment:
We need to handle the restriction that was added at some point during spec dev where minLod must be less-or-equal to last level accessed. We can replace the view with NULL descriptor in this case.

Reply:
Done,